json-endpoints/class.wpcom-json-api-post-endpoint.php

```diff
@@ -449,7 +449,10 @@ function get_post_by( $field, $field_value, $context = 'display' ) {
 						$show = true;
 
 					// Only business plan subscribers can view custom meta description.
-					if ( Jetpack_SEO_Posts::DESCRIPTION_META_KEY == $meta->key && ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+					if (
+						Jetpack_SEO_Posts::DESCRIPTION_META_KEY == $meta['meta_key']
```

Comment:
Should this be `===` for strict equality?  PHP can be funny (bad funny) sometimes w/r/t type juggling in equality.